web/packages/core/src/shadow-template.ts

```diff
@@ -228,3 +232,86 @@ ruffleShadowTemplate.innerHTML = `
 
     <ul id="context-menu"></ul>
 `;
+
+export const preloaderTemplate = document.createElement("template");
```

Comment:
Is a separate shadow template really needed? Why not just adding this to the main `ruffleShadowTemplate`, hidden by default?